models/intermediate/shopify__emails__order_aggregates.sql

```diff
@@ -0,0 +1,33 @@
+with orders as (
+
+    select *
+    from {{ var('shopify_order') }}
+    where email is not null
+
+), transactions as (
+
+    select *
+    from {{ ref('shopify__transactions' )}}
+    where lower(status) = 'success'
```

Comment:
I'm wondering if this where condition should be configurable? It makes sense in most cases for this to hold true but maybe customers will sometimes want to have transactions in other states too count toward their order definitions. Probably not needed now but could be a good feature request to file for later.

Reply:
yeah maybe for later, as i totally copied this logic from the pre-existing `shopify__customers__order_aggregates` model

Reply:
the other values for status are pending, failure, and error, so i think it still makes sense to limit to successful transactions for this particular metric. i could however see a separate metric around "failed transactions" being valuable

Reply:
i'll note for my "enhance existing models" task to perhaps add such metrics